libraries/hyperhtml/src/component-tests.js

```diff
@@ -121,16 +121,22 @@ describe('attributes and properties', function() {
     expect(data).toEqual('hyperHTML');
   });
 
-  it('will pass array data as a property', function() {
+  it('will pass array data as a property', function(done) {
```

Comment:
In other tests we've turned this into an `async function` and used `await Promise.resolve()` to wait for the next microtask. If that would work in this instance, would you be ok updating the test to use that approach for consistency?

Reply:
done